kodilla-patterns/src/main/java/com/kodilla/patterns/prototype/library/Library.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.patterns.prototype.library;
+
+import com.kodilla.patterns.prototype.TasksList;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class Library extends Prototype {
+    private String name;
+    private Set<Book> books = new HashSet<>();
+
+    public Library(final String name) {
+        this.name = name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Set<Book> getBooks() {
```

Comment:
Trochę dziwne że nie ukrywasz implementacji na HashSecie tworząc metody `addBook` i `removeBook`, tylko pozwalasz na bezpośredni dostęp do wewnętrznej kolekcji. Powiedzmy, że z punktu widzenia wzoraca prorotype nie ma to znaczenia, ale ogólnie byłoby lepiej.